docs/configuration-options/index.md

```diff
@@ -2269,6 +2269,16 @@ These options reflect new features that have not yet been fully finalized. Avail
 
 Determines after how many runs cached assets that are no longer used by plugins should be removed.
 
+### experimentalDeepDynamicChunkOptimization
+
+|  |  |
+| --: | :-- |
+| Type: | `number` |
```

Comment:
`boolean`?

Reply:
Ooops. Copy-and-paste, no time to waste 🙄